tutorials/lander.md

```diff
@@ -0,0 +1,422 @@
+\page lander
+
+# Overview
+
+This tutorial explains how to create and load an underwater lander vehicle in
+Gazebo. This type of vehicles don't usually have control surfaces or thrusters.
+They typically sink until they reach the seafloor or certain depth and go back
+to the surface while they collect sensor measurements.
+
+## Related tutorials
+
+https://gazebosim.org/api/sim/8/create_vehicle.html
+
+https://gazebosim.org/api/sim/8/adding_visuals.html
+
+https://gazebosim.org/api/sim/8/frame_reference.html
+
+https://gazebosim.org/api/sim/8/adding_system_plugins.html
+
+https://gazebosim.org/api/sim/8/theory_buoyancy.html
+
+https://gazebosim.org/api/sim/8/theory_hydrodynamics.html
+
+# Create your vehicle
+
+As an example, we'll go through the process of creating an Inkfish underwater
+lander vehicle.
+
+## Basic SDF
+
+Create a workspace to store your brand new model named `my_lander`.
+
+```bash
+mkdir -p ~/gazebo_maritime/models/my_lander && cd ~/gazebo_maritime/models/my_lander
+```
+
+Download the `model.config` file and copy it within that directory:
+
+```bash
+wget https://raw.githubusercontent.com/gazebosim/gz-sim/main/tutorials/files/lander/model.config -o ~/gazebo_maritime/models/my_lander/model.config
+```
+
+In its simple version, the lander does not have any moving pieces, so the SDF
+model only has one single link, `base_link` in our case.
+
+Create a `model.sdf` and paste the following content:
+
+```xml
+<?xml version="1.0"?>
+<sdf version="1.6">
+  <model name="lander">
+    <link name="base_link">
+      <inertial>
+        <!-- Center of mass -->
+        <pose>0 0 -0.982494 0 0 0</pose>
+
+        <mass>1209.175</mass>
+        <inertia>
+          <ixx>205.227945746</ixx>
+          <ixy>0</ixy>
+          <ixz>0</ixz>
+          <iyy>412.254844006</iyy>
+          <iyz>0</iyz>
+          <izz>383.659583213</izz>
+        </inertia>
+      </inertial>
+
+      <visual name="chassis_visual">
+        <geometry>
+          <box>
+            <size>1.325 1.1503 1.5</size>
+          </box>
+        </geometry>
+      </visual>
+    </link>
+
+  </model>
+</sdf>
+
```

Comment:
```suggestion
</sdf>
```

Reply:
Line removed in [2648ff4](https://github.com/gazebosim/gz-sim/pull/2259/commits/2648ff4f45e479dc7636bb6ff6cce1f4db6e2b0e).